docs/resources/pumls/frame.puml

```diff
@@ -0,0 +1,38 @@
+@startuml Frame
+
+!includeurl ./style.puml
+
+package "Frame" {
+
+    abstract class FrameCipherBase<<abstract>> {
+        + {abstract} encrypt()
+        + {abstract} decrypt()
+    }
+
+    class QATFrameCipher {
+        + encrypt()
+        + decrypt()
+    }
+
+    class Frame {
+        - provider: StreamProvider
+        - pipeline_id: str
+        - sequence: int
+        - raw: np.ndarray
+        - ts_new: float
+        - ts_infer_start: float
+        --
+        + {static} get_sequence(): int
```

Comment:
This function is `classmethod`, is there `classmethod` in plantUML?

Reply:
no, it can be delcared as static.